daemon/volumes.go

```diff
@@ -119,7 +119,15 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 	}
 
-	// 3. Read bind mounts
+	// 3. Read tmpfs
+	for destination := range hostConfig.Tmpfs {
+		mountPoints[destination] = &volume.MountPoint{
+			Source:      "tmpfs",
```

Comment:
Hmm, I'm not sure about using `Source` for specifying that it is a tmpfs.
See #22373 which introduces "Type", we have a "bind" type and a "volume type. Could be a "tmpfs" type?